storybook/src/Storybook/BiometricsPopup.qml

```diff
@@ -80,16 +75,5 @@ Dialog {
                 root.obtainingPasswordSuccess(root.selectedProfileIsKeycard ? root.pin : root.password)
             }
         }
-        StatusButton {
-            Layout.fillWidth: true
-            Layout.alignment: Qt.AlignHCenter
-            type: StatusBaseButton.Type.Danger
-            focusPolicy: Qt.NoFocus
-            text: "Simulate wrong fingerprint"
-            onClicked: {
-                root.close()
-                root.obtainingPasswordError("Wrong fingerprint provided.", Constants.keychain.errorType.keychain, true)
```

Comment:
There's no such behaviour in `MacOS`. When using the unregistered fingerprint, you'll be just given an attempt to try again. After 3 attempts you're asked for a password.

Anyway, I thought we don't want to cover the case when the used failed way too many times. Or do we? 🤔

Reply:
So, do we return error at some point or the sure can try infinitely? If yes we need only cancel, right?

Reply:
So it's either cancel or success.
There will be another case when we implement this: https://github.com/status-im/status-desktop/pull/17166#discussion_r1939220420, but that's a different story.